Rakefile

```diff
@@ -347,13 +347,13 @@ end
 task :setup => ["git:clone", "bundle:install"]
 
 task :default do
-  run_command "bin/rake"
+  run_command "bundle exec rake"
 end
 
 desc "publish cukes to relishapp.com"
 task :relish, :version do |_, args|
   raise "rake relish[VERSION]" unless args[:version]
-  run_command "bin/rake relish['#{args[:version]}']", :except => ['rspec']
+  run_command "bundle exec rake relish['#{args[:version]}']", :except => ['rspec']
```

Comment:
Did windows have problems with `bin/rake` for some reason?

FWIW, most of these tasks are things that only I use (e.g. to release new versions, update the relish docs, etc) and I prefer the binstubs approach.  I'd prefer to stick with `bin/rake`.  If windows needs it to be `bundle exec`, maybe we can put something like this at the start of the rake file?

``` ruby
RAKE_COMMAND = if RbConfig::CONFIG['host_os'] =~ /cygwin|mswin|mingw|bccwin|wince|emx/
  "bundle exec rake"
else
  "bin/rake"
end
```

...and then we can use `RAKE_COMMAND` in all these places -- that way, it'll use `bundle exec rake` on windows, and `bin/rake` elsewhere.